charts/openmetadata/templates/deployment.yaml

```diff
@@ -2,21 +2,52 @@ apiVersion: apps/v1
 kind: Deployment
 metadata:
   name: {{ include "OpenMetadata.fullname" . }}
+  annotations:
+    {{- if .Values.annotations }}
+    {{- range $key, $val := fromYaml .Values.annotations }}
+    {{ $key }}: {{ $val | quote }}
+    {{- end}}
+    {{- end}}
   labels:
-    {{- include "OpenMetadata.labels" . | nindent 4 }}
+    app.kubernetes.io/name: '{{ include "OpenMetadata.name" .}}'
```

Comment:
I would recommend using [helper class](https://github.com/open-metadata/openmetadata-helm-charts/blob/fd71bcc9eab12eebc13296f7b3f4a3a1f98dd392/charts/openmetadata/templates/_helpers.tpl) to define labels starting with `app.kubernetes.io/*`!

Reply:
used helper class to define labels starting with app.kubernetes.io/*